.travis.yml

```diff
@@ -24,4 +25,4 @@ matrix:
 install:
   - rustup component add rustfmt
 script:
-  - cargo fmt -- --check
\ No newline at end of file
+  - cargo fmt -- --check
```

Comment:
The addition of the newline ended up in the wrong commit. Please move it to the first commit.